packages/schema/parse/src/__tests__/cases/sanity/output.ts

```diff
@@ -86,6 +94,27 @@ export const output: TypeInfo = {
             createScalarPropertyDefinition("arg", "String", true)
           ],
           return: createScalarPropertyDefinition("queryMethod", "Int", true)
+        },
+        {
+          ...createMethodDefinition("query", "importedObjectMethod"),
+          arguments: [
+            createObjectPropertyDefinition(
```

Comment:
Do I need uri and namespace here for imported objects or I can just treat as regular user object?

Reply:
The UI & namespace are available on the TestImport_Object definition, so no need to carry those properties through to this instance.

Reply:
So in short, all import metadata is only needed on the ImortedObjectDefinition, and not on the arguments + return type definitions.